packages/blob-storage-manager/src/BlobStorageManager.ts

```diff
@@ -160,11 +171,12 @@ export class BlobStorageManager<
         },
       },
       async (span) => {
-        const availableStorages = Object.entries(this.#blobStorages).filter(
-          ([, storage]) => storage
-        ) as [SNames, BlobStorage][];
+        // If no storages are provided, use all the storages
```

Comment:
Why? Usually you would expect that if no storages are provided, then none is used.

Reply:
This refers to the selected storages provided by the user.

I added a new parameter to the `storeBlob` operation that allows the user to provide specific storages where they want to store the blobs in. If no storage is provided we default to the normal behaviour which is trying to store the blob in all the storages. 

Maybe it should be better to rename the variable to `selectedStorages` to convey better this functionality